src/compressed_tensors/utils/offload.py

```diff
@@ -341,6 +343,42 @@ def delete_from_weights_map(
         )
 
 
+@contextlib.contextmanager
+def disable_offload(module: torch.nn.Module):
+    """
+    Disable module onloading and offloading. Parameters will stay on their current
+    device
+
+    :param module: module to disable offloading for
+    """
+    if has_offloaded_params(module):
+        module._hf_hook.offload = False
+        yield
+        module._hf_hook.offload = True
+    else:
+        yield
+
+
+@contextlib.contextmanager
+def align_modules(
+    modules: Union[torch.nn.Module, Iterable[torch.nn.Module]],
+    execution_device: Optional[torch.device] = None,
+):
+    """
+    Onload modules to a device, disable onload attempts triggered by forward calls
```

Comment:
should it be disbale offload attempts?

Reply:
Also could we improve the docstring slightly to say that this is a context manager?